http/jsonapi/runtime/standart_params.go

```diff
@@ -4,24 +4,46 @@
 package runtime
 
 import (
+	"fmt"
 	"net/http"
 	"strconv"
 	"strings"
+
+	"github.com/caarlos0/env"
+	"github.com/pace/bricks/maintenance/log"
 )
 
 // QueryOption is a function that applies an option (like sorting, filter or pagination) to a database query
 type QueryOption func(query Query) Query
 
 // Query based on orm.Query but only the needed function
 // guaranties testability of all of the following functions
+// An Implementation MUST prepare statements to prevent SQL injection
```

Comment:
We can't use prepared statements as we are using PGBouncer with transaction mase isolation, is this really required?